cmd/rofl/mgmt.go

```diff
@@ -112,6 +114,18 @@ var (
 				cobra.CheckErr(fmt.Errorf("failed to write manifest: %w", err))
 			}
 
+			// Initialize git repository if not already present.
+			_, err = os.Stat(".git")
+			if err != nil && errors.Is(err, os.ErrNotExist) {
+				// Git directory doesn't exist, try to initialize.
+				gitInitCmd := exec.Command("git", "init")
+				if err = gitInitCmd.Run(); err != nil {
+					fmt.Printf("Git repository not initialized: %s.\n", err)
+				} else {
+					fmt.Printf("Git repository initialized.\n")
+				}
+			}
+
```

Comment:
Please also append `*.orc` to `.gitignore`, or create a new file if it doesn't exist yet. (regardless of whether git init succeeded or not.)

And maybe move the git-related code into a helper function not to clutter the init command.